app/schemas/data-operation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.14.0] - 2022-02-19
+### Added
+- DataOperation.point - Spatial point provided by the CMR
```

Comment:
I believe this is the spatial point specified by the user. Also it is now under `subset.point`.

Reply:
Ah yes ... good catch!

Reply:
Done.